basic-javascript/project-1.spec.js

```diff
@@ -1,3 +1,182 @@
 const helpers = require('./project-1');
 
-// start testing!
+describe('multiplyByTen', () => {
+  test('Correct multiplication', () => {
+    const actual = helpers.multiplyByTen(2);
+    expect(actual).toEqual(20);
+  });
+  test('By NaN', () => {
+    const actual = helpers.multiplyByTen(NaN);
+    expect(actual).toEqual(NaN);
+  });
+  test('By a number in String format', () => {
+    const actual = helpers.multiplyByTen('2');
+    expect(actual).toEqual(20);
+  });
+  test('By a String other than a digit', () => {
+    const actual = helpers.multiplyByTen('hola');
+    expect(actual).toEqual(NaN);
```

Comment:
I'm curious - what made you decide not to do these inline? Like so: `expect(helpers.multiplyByTen(undefined)).toBeNaN();`

Reply:
just testing how 'jest' performs outside its default helpers.

Reply:
Ok cool.